<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leaderboard</class>
 <widget class="QDialog" name="leaderboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	
	image: url(:/new/prefix1/download-removebg-preview.png);
    background-color: rgb(34, 128, 168);
    color: rgb(255, 255, 255);
    padding: 5px 9px;
    border-radius: 15px;
}

QPushButton:hover {
    background-color: rgb(33, 138, 45);
    color: rgb(255, 255, 255);
}

QPushButton:pressed {
    background-color: rgb(8, 74, 41);
    color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100000</width>
     <height>10000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>50</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 50pt &quot;Yu Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>GAME STATS</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>581</width>
     <height>186</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="gamesPlayedLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>50</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 50pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="winCountLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>50</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 50pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentStreakLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>50</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 50pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxStreakLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>50</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 50pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="gamesPlayedLabel_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 16pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>Games
Played</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="winCountLabel_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 16pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>Wins</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentStreakLabel_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 16pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>Current
Streak</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxStreakLabel_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>100000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>16</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 16pt &quot;Yu Gothic&quot;;</string>
        </property>
        <property name="text">
         <string>Max
Streak</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
